libvast/vast/plugin.hpp

```diff
@@ -44,6 +44,14 @@ extern "C" struct plugin_version {
   uint16_t minor;
   uint16_t patch;
   uint16_t tweak;
+
+  /// An additional field containing the version of the libvast that is linked
+  /// against. Should compare equal to VAST_VERSION.
+  const char* libvast_version;
+
+  /// An additional field containing the hash of the build tree of the libvast
+  /// that is linked against. Should compare equal to VAST_BUILD_TREE_HASH.
+  const char* libvast_build_tree_hash;
 };
```

Comment:
These should not go into `plugin_version`, I think. They are conceptually different things and I don't think we need to add them in this particular struct. Can't these just be tracked separate from `plugin_version`?

Reply:
Then you'll need to add additional exported functions to the plugin API.

Reply:
Seems cleaner to me that way, now that you mention it.